src/main/java/org/spongepowered/mod/mixin/core/event/player/MixinEventPlayer.java

```diff
@@ -45,19 +46,7 @@ public MixinEventPlayer(EntityLivingBase entity) {
     }
 
     @Override
-    public Player getPlayer() {
-        return (Player) this.entityPlayer;
-    }
-
-    @Override
-    public Player getLiving() {
-        return (Player) this.entityPlayer;
-    }
-
-    @Override
-    public Player getHuman() {
-        return (Player) this.entityPlayer;
-    }
+    public Living getLiving() {return (Player) this.entityPlayer;}
```

Comment:
I think that you need to make this `public Player getLiving()`, since that is what is required by the interface PlayerEvent. The same follows for other methods.